consensus/bor/bor.go

```diff
@@ -773,7 +773,7 @@ func (c *Bor) Seal(chain consensus.ChainHeaderReader, block *types.Block, result
 	// Sweet, the protocol permits us to sign the block, wait for our time
 	delay := time.Unix(int64(header.Time), 0).Sub(time.Now()) // nolint: gosimple
 	// wiggle was already accounted for in header.Time, this is just for logging
-	wiggle := time.Duration(successionNumber) * time.Duration(c.config.BackupMultiplier) * time.Second
+	wiggle := time.Duration(successionNumber) * time.Duration(c.config.CalculateBackupMultiplier(number)) * time.Second
```

Comment:
Does the London fork also introduces changes in the consensus?

Reply:
backup multiplier is introduced by us
will be increasing it to 4 from London fork

Reply:
What does this mean? It seems quite a big of change if it changes the consensus protocol to be left undocumented.